cmd/openshift-install/agent/waitfor.go

```diff
@@ -18,17 +18,40 @@ func NewWaitForCmd() *cobra.Command {
 		},
 	}
 
+	cmd.AddCommand(newWaitForBootstrapCompleteCmd())
 	cmd.AddCommand(newWaitForInstallCompleteCmd())
 	return cmd
 }
 
+func newWaitForBootstrapCompleteCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "bootstrap-complete",
+		Short: "Wait until the cluster bootstrap is complete",
+		Args:  cobra.ExactArgs(0),
+		Run: func(cmd *cobra.Command, args []string) {
+			assetDir := cmd.Flags().Lookup("dir").Value.String()
+			if len(assetDir) == 0 {
+				logrus.Fatal("No cluster installation directory found")
+			}
+			err := runWaitForBootstrapCompleteCmd(assetDir)
+			if err != nil {
+				logrus.Fatal(err)
```

Comment:
In case of bootstrap failure it could be useful to log the cluster operators status via `logClusterOperatorConditions` and return the same error code as in https://github.com/openshift/installer/blob/e598e161c7142816fbfa39196d4f857047f1baab/cmd/openshift-install/waitfor.go#L49-L59

Reply:
I've added some verbosity to this yesterday using the logClusterOperatorConditions()